.circleci/config.yml

```diff
@@ -172,6 +172,51 @@ jobs:
             docker login -u "$DOCKER_USER" -p "$DOCKER_PASS"
             make -C ./docker version=${CIRCLE_TAG} alias=latest push
 
+  publish-docker-image-arm64:
+    machine:
+      image: ubuntu-2004:current
+    resource_class: arm.medium
```

Comment:
This would configure the job to run in Arm environment https://circleci.com/docs/using-arm/

This would make building an Arm image considerably easier, as I'm unsure if the usual amd64 CircleCI images have qemu setup to allow building different platform images.

Reply:
Seems reasonable